xbps-src

```diff
@@ -394,7 +394,7 @@ umask 022
 #
 # Required utilities in host system for the bootstrap target.
 readonly REQHOST_UTILS_BOOTSTRAP="file objdump find make gawk bash sed gcc g++ gnat \
-    perl bsdtar gzip patch flock pkg-config"
+    perl bsdtar gzip patch flock pkg-config makeinfo"
```

Comment:
this is one is unnecessary because we don't require texinfo in the host system.

Reply:
Yup.